src/libs/listenToStorageEvents/index.js

```diff
@@ -10,7 +12,7 @@ function listenToStorageEvents(callback) {
         try {
             newValue = JSON.parse(e.newValue);
         } catch (err) {
-            console.debug('Could not parse the newValue of the storage event', err, e);
+            Log.hmmm('Could not parse the newValue of the storage event', {event: e, error: err});
```

Comment:
Not sure but the event could have all sorts of information. Maybe we should only log the `e.key` and not it's value.